Lobe_iOS/PlayViewModel.swift

```diff
@@ -1,75 +1,87 @@
 //
-//  PlayViewModel.swift
-//  Lobe_iOS
-//
-//  Created by Elliot Boschwitz on 12/1/20.
 //  Copyright © 2020 Microsoft. All rights reserved.
 //
 
 import Combine
 import SwiftUI
 
 enum PlayViewMode {
-    case Camera
-    case ImagePreview
-    case NotLoaded
+  case Camera
+  case ImagePreview
+  case NotLoaded
 }
 
 /// View model for the Play View
 class PlayViewModel: ObservableObject {
-    @Published var classificationLabel: String?
-    @Published var confidence: Float?
-    @Published var viewMode: PlayViewMode = PlayViewMode.NotLoaded
-    @Published var showImagePicker: Bool = false
-    @Published var imageFromPhotoPicker: UIImage?
-    var captureSessionManager: CaptureSessionManager
-    let project: Project
-    var imagePredicter: PredictionLayer
-    private var disposables = Set<AnyCancellable>()
+  @Published var classificationLabels: [String]?
+  @Published var confidences: [Float]?
+  
+  @Published var firstLabel: String?
```

Comment:
Good changes here. I think I'll create new structs which combine label and confidence as well as a class for pointing to our top 3 results, rather than declaring each one individually.

Reply:
THANK YOU for this! I didn't know how to do it properly in SwiftUI. Love your suggestion.